public/app/AppWrapper.tsx

```diff
@@ -107,18 +108,20 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                 >
                   <ModalsProvider>
                     <GlobalStyles />
-                    {config.featureToggles.commandPalette && <CommandPalette />}
+                    {!isDashboardPubliclyViewed() && config.featureToggles.commandPalette && <CommandPalette />}
```

Comment:
Unsure of the naming here. Also don't really like littering this component with these "are we looking at public page" checks.

Reply:
How about `isPublicDashboardView`